src/Entity/CommitteeMembership.php

```diff
@@ -19,13 +20,21 @@
  *         @ORM\UniqueConstraint(
  *             name="adherent_has_joined_committee",
  *             columns={"adherent_id", "committee_id"}
+ *         ),
+ *         @ORM\UniqueConstraint(
+ *             name="adherent_votes_in_committee",
+ *             columns={"adherent_id", "enable_vote"}
  *         )
  *     },
  *     indexes={
  *         @ORM\Index(name="committees_memberships_role_idx", columns="privilege")
  *     }
  * )
  * @ORM\Entity(repositoryClass="AppBundle\Repository\CommitteeMembershipRepository")
+ * @UniqueEntity(
```

Comment:
Even if this validation is right: it is **now** useless.

Please remove this as it is not used.